src/platform/Darwin/CHIPPlatformConfig.h

```diff
@@ -66,5 +75,18 @@ extern "C" int __cxa_atexit(void (*f)(void *), void * p, void * d);
 #endif // CHIP_CONFIG_BDX_MAX_NUM_TRANSFERS
 
 #ifndef CHIP_CONFIG_KVS_PATH
+#if TARGET_OS_IPHONE
+#define CHIP_CONFIG_KVS_PATH "chip.store"
```

Comment:
Shouldn't this be for all the cases that are not TARGET_OS_OSX or something?

Reply:
> When does this actually take effect? The default KVS isn't used from the Framework right? Also, what's the effect of having a relative paths here?

I assumed it might be used via `chip-tool`.